.drone.star

```diff
@@ -2196,6 +2196,7 @@ def ocisService():
                 "STORAGE_USERS_DRIVER_OWNCLOUD_DATADIR": "/srv/app/tmp/ocis/owncloud/data",
                 "WEB_ASSET_PATH": "%s/dist" % dir["web"],
                 "WEB_UI_CONFIG": "/srv/config/drone/config-ocis.json",
+                "FRONTEND_ENABLE_RESHARING": "true",
```

Comment:
Can you add this in `docker-compose.yml` too? Thanks!

Reply:
Valid question

Reply:
Took care myself, thanks for the hint